docs/_future/deploy_your_app.md

```diff
@@ -7,10 +7,212 @@ layout: tutorial
 permalink: /future/deploy-your-app
 ---
 
-# Deploy your app <span class="label-beta">BETA</span>
+# Deploy your app to Heroku<span class="label-beta">BETA</span>
```

Comment:
what are your thoughts on keeping this as "Deploy your app" for now? I imagine once we have Run on Slack capabilities for Bolt this page will likely evolve and the Heroku guide may be transitioned elsewhere

Reply:
I have no objections! Always a fan of forward thinking 👏